sktime/param_est/base.py

```diff
@@ -4,8 +4,8 @@
     class name: BaseParamFitter
 
 Scitype defining methods:
-    fitting                - fit(X)
-    updating               - update(X)
+    fitting                - fit(X, y)
+    updating               - update(y, X=None)
```

Comment:
X is mandatory, y is optional

Reply:
Made the changes in the docstring, and the function is doing exactly what you mentioned above.